addons/wp/stringtable.xml

```diff
@@ -427,6 +433,15 @@
             <English>Hit</English>
             <German>Getroffen</German>
         </Key>
+        <Key ID="STR_Lambs_WP_HOLD">
+            <English>Hold</English>
```

Comment:
we may should look if we find this in the vanilla string tables we don't need those duplicated strings. this can be done after Merge and be done with the Unify stringtables later

Reply:
I agree.